services/login_service/app/auth/keycloak_admin_token.py

```diff
@@ -0,0 +1,38 @@
+import requests
+from config.settings import Config
+import os
+
+cert_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "../server.crt"))
+
+
+def get_admin_token():
+    url = (
+        "https://35.190.204.129:8443/realms/login_signup/protocol/openid-connect/token"
+    )
+
+    data = {
+        "client_id": Config.KEYCLOAK_CLIENT_ID,
```

Comment:
You can use `Config.KEYCLOAK_SERVER_URL` also here, so this url is easly modifiable